packages/web/src/components/card-legacy/desktop/CollectionArtCard.tsx

```diff
@@ -144,8 +146,8 @@ const CollectionArtCard = g(
       playlistName: playlist_name,
       isOwner: currentUserId === user_id,
       includeShare: true,
-      includeRepost: true,
-      includeSave: true,
```

Comment:
Note that this was broken before because the prop had been renamed to `includeFavorite` BUT we cast through `as unknown as CollectionMenuProps` here so we weren't warned about it